src/zac/conf/local_example.py

```diff
@@ -1,9 +1,10 @@
 """
 Any machine specific settings when using development settings.
 """
-
 import os
 
+from .settings import *
```

Comment:
bad practice - often leads to circular imports. Usually you import just what you need, I'd leave this line out completely

Reply:
removed import and placed DB config to `settings.py`